static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -149,7 +149,38 @@ $renaultLight: #fff9e9;
     vertical-align: top;
 
     .vjs-big-play-button .vjs-control-text {
-        display: none;
+        margin-left: -40px;
+        margin-top: -68px;
+        width: 136px;
+        height: 48px;
+        text-indent: -999px;
+
+        @include mq(tablet) {
+            margin-left: -70px;
+            margin-top: -25px;
+        }
+
+        &:before {
+            content: 'Watch now';
+            background-color: unset;
```

Comment:
Yeah it's CSS3/4 stuff
https://drafts.csswg.org/css-cascade-3/#inherit-initial

On Wed, 1 Jun 2016 at 11:22 Zofia Korcz <notifications@github.com> wrote:

> In static/src/stylesheets/module/commercial/glabs/_hosted.scss
> <https://github.com/guardian/frontend/pull/13133#discussion_r65336235>:
>
> > @@ -149,7 +149,38 @@ $renaultLight: #fff9e9;
> >      vertical-align: top;
> >
> >      .vjs-big-play-button .vjs-control-text {
> > -        display: none;
> > +        margin-left: -40px;
> > +        margin-top: -68px;
> > +        width: 136px;
> > +        height: 48px;
> > +        text-indent: -999px;
> > +
> > +        @include mq(tablet) {
> > +            margin-left: -70px;
> > +            margin-top: -25px;
> > +        }
> > +
> > +        &:before {
> > +            content: 'Watch now';
> > +            background-color: unset;
>
> It works fine at least in Chrome and Firefox. But I don't know why it's
> not mentioned in the spec (https://www.w3.org/TR/CSS2/colors.html#q14.0).
> The MDN (https://developer.mozilla.org/en-US/docs/Web/CSS/background-color)
> says that the initial value is transparent so maybe it would be safer to
> use this one.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/guardian/frontend/pull/13133/files/f504c98835a556d37658b9e8f846edb7528ed3c3#r65336235>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAmc2BwQM6yo7CbQqD0-hNCvHrt-Sey-ks5qHV1QgaJpZM4Iqp94>
> .
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396